aws-datastore/src/test/java/com/amplifyframework/datastore/DataStoreConfigurationTest.java

```diff
@@ -96,11 +100,13 @@ public void testDefaultOverriddenFromConfigurationAndObject() throws JSONExcepti
         DummyConflictHandler dummyConflictHandler = new DummyConflictHandler();
         DataStoreErrorHandler errorHandler = DefaultDataStoreErrorHandler.instance();
 
+        DataStoreSyncExpression syncExpression = () -> BlogOwner.ID.beginsWith("a");
```

Comment:
The `RandomString.string()` would have the same behavior here, but it reads like "oh, the value of the ID isn't important for the test, so long as it's there."